sarahbenmessaoud.svg

```diff
@@ -0,0 +1,106 @@
+<svg viewBox="0 -25 400 200" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You’re uploading your SVG to the root of our project. Could you just remove this file?